### ISSUE TITLE
Suggestion: Easier make simple API bindings

### ISSUE THREAD

**Issue by EmilMoe** [JosephusPaye/Keen-UI#79]
How would you think about the ability to pass an URL as options, so that I can just reference an API endpoint that populates my select? It could maybe be done so when you pass `options` as a String?

It would also be nice with some props where I can define `text` and `value` as different keys.

~~~~html
<ui-select
  options="http://api.domain.com/v1/users"
  :keys="{ text: 'name', value: 'id'}">
</ui-select>
~~~~

This might also apply to other components.

**Comment by JosephusPaye** [comment]
I think adding a prop to redefine the keys for `text` and `value` is a good idea and it will be available in the next version. 

However, I think passing a URL for the component to make an HTTP is request is a bit out of scope for Keen UI (even though it would be convenient).

**Comment by EmilMoe** [comment]
I agree it's a bit out of the scope. I tried to think if there was a way to make an extension so it's optional, but I couldn't come up with a pattern for that. I don't want to make a whole collection of components wrapping your components, that's more work that it's worth. But I don't know if there's a way to create an extension that adds functionality to existing components.

**Comment by JosephusPaye** [comment]
Implemented in https://github.com/JosephusPaye/Keen-UI/commit/5cd6be33411e05fa2ac03cb34f2ed998b6c8fdc4

**Comment by JosephusPaye** [comment]
The `keys` prop has landed in [v0.8.9](https://github.com/JosephusPaye/Keen-UI/releases/tag/v0.8.9). It works on both [UiSelect](https://josephuspaye.github.io/Keen-UI/#/ui-select-docs) and [UiAutocomplete](https://josephuspaye.github.io/Keen-UI/#/ui-autocomplete-docs).
